pylab/website/templates/website/project_details.html

```diff
@@ -28,4 +30,37 @@
   {{ project.status_description|capfirst }}</p>
 {% endif %}
 
+{% if user.is_authenticated %}
+  {% get_comment_form for project as form %}
```

Comment:
Would be great to have a test for the comment form. We don't have to test external modules functionality, but module's integration into ```project-details``` page can be tested.